### PR TITLE
Atualização da versão em inglês

### PR DESCRIPTION
**Pull request by francilioaraujo**

### CODE REVIEW

**Comment by francilioaraujo** [comment]
This PR updates the english version and makes some updates to the text so the travis build completes successfully.

Este PR atualiza a versão em inglês e atualiza algumas partes do texto para que o build do travis complete com sucesso.

File: book/04-git-server/sections/gitlab.asc [hunk 1]
```diff
@@ -38,7 +38,7 @@ image::images/gitlab-menu.png[O item Área de administração no menu GitLab.]
 Usuários no GitLab são contas que correspondem a pessoas.
 As contas de usuário não têm muita complexidade; A conta de usuário é uma coleção de informações pessoais anexadas aos dados de login.
 Cada conta de usuário vem com um *namespace*, que é um agrupamento lógico de projetos que pertencem a esse usuário.
-Se o usuário +jane+ tivesse um projeto chamado +project+, o URL do projeto seria http://servidor/jane/project[].
+Se o usuário +jane+ tivesse um projeto chamado +project+, o URL do projeto seria `http://servidor/jane/project`[].
```
Comment (jnavila) [inline book/04-git-server/sections/gitlab.asc:41]: You can remove the trailing square brackets. They are not needed.

**Review by jnavila — COMMENTED** [review]
You also need to remove the link to stack overflow at the end of the jgit section.

**Comment by carlosschults** [comment]
@francilioaraujo Obrigado pela ajuda!

**Comment by francilioaraujo** [comment]
> You also need to remove the link to stack overflow at the end of the jgit section.

The link was removed from the translation

File: book/04-git-server/sections/gitlab.asc (hunk 1 above)
Comment (francilioaraujo) [reply book/04-git-server/sections/gitlab.asc:41]: I will address this in a future PR. Thank you

**Pull request merged**